File: Searching/linerar_search.py
def linerar_search(a,target):
    for i in range(len(a)):
        if a[i] == target:
            return i
    return -1
a=[1,2,3,4,5]
target=6
index=linerar_search(a,target)
if(index==-1):
    print("not Found")
else:
    print("found at index",index)

File: Searching/linerar_search.go
package main

import "fmt"

func linear_search(a [7]int, t int) int{
	left,right:=0,len(a)-1
    for left <=right{
        mid:=(left+right)//2
        if (a[mid] == t){
            return mid
		}else if (a[mid] <t){
                left = mid+1
       } else{
                right = mid-1
		}
	}
	return -1
}
func main() {
	a := [7]int{1, 2, 3, 4, 5,7,9}
	fmt.Println(a)
	target := 7
	index:=linear_search(a, target)
	if(index==-1){
	    fmt.Println("not Found")
	}else {
	    fmt.Println("found at index",index)
	}
}
